### PR TITLE
Add: Functionality to Cart and Wishlist 

### PR DESCRIPTION
**Pull request by devsurajrai**
Added the relevant functionality to Cart and Wishlist without auth. Please review and suggest any changes you may feel.

Check it out live: https://rococo-pasca-e1cb50.netlify.app/

Screenshot 2022-03-28 at 8 47 31 PM
Screenshot 2022-03-28 at 8 47 12 PM

### CODE REVIEW

File: src/components/ProductListingMain.component/product-listing-main.css [hunk 1]
```diff
@@ -12,7 +12,6 @@
     grid-template-columns: minmax(0, 1fr) minmax(0, 5fr);
     grid-template-rows: repeat(minmax(0, 1fr));
     grid-column-gap: 1rem;
-    column-gap: 1rem;
 
-    overflow-y: hidden;
+    /* overflow-y:hidden; */
```
Comment (palash23jain) [inline src/components/ProductListingMain.component/product-listing-main.css:16]: commented code can be removed

**Review by palash23jain — APPROVED** [review]
readable code

File: src/components/CartMain.component/CartProductCard.component/CartProductCard.jsx [hunk 2]
```diff
@@ -1,38 +1,94 @@
...
-              <button class="qty-btn">+</button>
+            <span className="text-sm fw-b color-grey ">
+              {discountOnProduct}% Off
+            </span>
+            <div className="flex-r align-items-center m-b-xs">
+              <span>Quantity:</span>{" "}
+              <button
+                className="qty-btn"
+                onClick={() =>
+                  cartWishlistStateDispatch({
```
Comment (varnita619) [inline src/components/CartMain.component/CartProductCard.component/CartProductCard.jsx:44]: good naming convension

File: src/components/CartMain.component/cart-main.css [hunk 3]
```diff
@@ -35,4 +35,11 @@
...
     background-color: white;
     padding: 3rem;
+  }
+
+  /* Empty Cart  */
+  .empty-cart{
+    display: flex;
+    justify-content:center;
+    align-items:center;
   }
```
Comment (varnita619) [inline src/components/CartMain.component/cart-main.css:45]: you can add new line at end of each file for clealiness

**Review by varnita619 — APPROVED** [review]
Rest all looks good

**Comment by saifudinbadani** [comment]
Checking the UI experience, it is working fine..  the remove from wishlist button can be replaced with heart icon.

**Pull request merged**